extensions/wikia/Email/Controller/FounderController.class.php

```diff
@@ -610,9 +610,10 @@ protected function getDetailsList() {
 		$discussionsEnabled =
 			\WikiFactory::getVarValueByName( 'wgEnableDiscussions', $this->wikiId );
 		if ( $discussionsEnabled ) {
+			$safeWikiUrl = rtrim( $this->wikiUrl, "/" );
```

Comment:
`safeWikiUrl` is a bit ambiguous. What's "safe" about it? Maybe `wikiUrlWithoutTrailingSlash` (or something more explicit like that but less wordy).